CONTRIBUTING.md

```diff
@@ -12,24 +12,23 @@ contributions.
 We encourage you to read this project's CONTRIBUTING policy (you are here), its
 [LICENSE](LICENSE.md), and its [README](README.md).
 
-## Getting Started
+# Getting Started
+## Building the Project
 
 For instructions on how to invoke the script, see the usage section of the [README](README.md)
```

Comment:
Let's switch the order of "Building the Project and "Building Dependencies" because first we install/build dependencies then build the project

Reply:
This is something we'll need to fix in the upstream templates in repo-scaffolder, but +1 Nat.